src/main/resources/logback.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <include resource="org/springframework/boot/logging/logback/base.xml" />
+    <logger name="org.springframework" level="INFO"/>
```

Comment:
this can be done using `application.properties` like `logging.level.org.springframework=info`, so no need in this file at all

Reply:
Replaced by the config in the yaml file